docs/S3_IAM_REST_API_&_CLI.md

```diff
@@ -0,0 +1,355 @@
+# Supported S3 IAM REST APIs and CLI Operations
```

Comment:
What about this:

Supported S3 IAM REST APIs and CLI operations in CORTX

Reply:
Done